sdk/src/main/scala/io/horizen/api/http/route/WalletBaseApiRoute.scala

```diff
@@ -44,9 +45,10 @@ abstract class WalletBaseApiRoute[
                                    sidechainNodeViewHolderRef: ActorRef,
                                    sidechainSecretsCompanion: SidechainSecretsCompanion
                         )(implicit val context: ActorRefFactory, override val ec: ExecutionContext, override val tag: ClassTag[NV])
-  extends SidechainApiRoute[TX, H, PM, FPI, NH, NS, NW, NP, NV] {
+  extends SidechainApiRoute[TX, H, PM, FPI, NH, NS, NW, NP, NV] with DisableApiRoute {
 
 
+  val myPathPrefix: String = "wallet"
```

Comment:
just a suggestion, discard if you think that's bad :)
rename myPathPrefix -> walletPathPrefix \ transactionPathPrefix \ blockPathPrefix \ etc.

Reply:
Ok for me, but what about something more generic, like routePathPrefix? Maybe in the future these fields will be moved to a base class.

Reply:
that's probably a different idea. What I was thinking is that this field is effectively a constant inside the actor, and adding some context into a variable name helps with readability - f.e. you would know right away what the value of walletPathPrefix is, without looking up where it is set.

Reply:
Ok, I see. I'll do as you suggested then.

Reply:
Done